russian/invite-friends/knock/knock.md

```diff
@@ -0,0 +1 @@
+Вы открыли дверь и отправились вместе с людьми в белых халатах в их замечательную машину из зефира с надписью "Санитары".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.